src/parser/mod.rs

```diff
@@ -8602,6 +8602,18 @@ impl<'a> Parser<'a> {
                 arg,
                 operator: FunctionArgOperator::Equals,
             })
+        } else if dialect_of!(self is DuckDbDialect) && self.peek_nth_token(1) == Token::Assignment
+        {
+            let name = self.parse_identifier(false)?;
+
+            self.expect_token(&Token::Assignment)?;
+            let arg = self.parse_wildcard_expr()?.into();
```

Comment:
is wildcard included?

Reply:
Probably not, thanks. To be honest, I just copied from the code few lines above which does the same for `=>` and `=`. Changing them to `parse_expr` doesn't break any test. Don't know if they need it too.
Anyway, i changed mine to `parse_expr`, thanks again 🙏